doc/Translation.md

```diff
@@ -0,0 +1,85 @@
+# Translation
+
+We need your help to translate Privacy Badger to every possible language.
+If you think you can help translating to your local language, here you
+will find instructions to help you in the process.
+
+At the moment we handle the translations of Privacy Badger with two different
+platforms: Transifex and GitHub. Feel free to use the tool you prefer.
+
+When translating you should always use the original (English) version as a 
+reference. You can also use existing translations from other languages to help
+you in case of doubt but you should always consider the English version as the correct one.
+
+## GitHub
+
+Translations on GitHub are done with JSON files.
+Each language has its own folder inside src/_locales (e.g. 'es' for Spanish,
+'ru' for Russian, etc.).
+Inside each of these folders there is a JSON file which contains the translated
+strings for that language. Each entry on the JSON file follows this structure:
+
+    "string_identifier": {
+        "message": "String text"
+    }
+    
+The translated string is the `"String text"` part and you should **NOT** change
+any other part of the entry.
+
+To contribute to the Privacy Badger translation through GitHub, first check the
+status of your local language translation: if you can't see a folder with your 
+local language code, the translation for that language is missing. In this case
```

Comment:
Should link "local language code" to https://developer.chrome.com/webstore/i18n?csw=1#localeTable.

Reply:
Ok